frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseResetBranchToRemoteAction.java

```diff
@@ -116,6 +122,20 @@ public void actionPerformed(AnActionEvent anActionEvent) {
       return;
     }
 
+    if (!PropertiesComponent.getInstance().getBoolean(RESET_INFO_SHOWN)) {
```

Comment:
`getBoolean` returns `false` (and doesn't e.g. throw an exception) if a key is missing? Maybe add a comment that states that explicitly